src/main/java/pages/HomeUnloggedPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import controls.Button;
+import core.Driver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class HomeUnloggedPage extends BasePage {
```

Comment:
I would suggest to combine this PO with HomeLoggedPage. just create a methos like isLoggedIn that returns a boolean.